Dispatcher/Views/ArticleCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ArticleCell.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 19/04/2023.
+//
+
+import UIKit
+
+class ArticleCell: UITableViewCell {
+    
+    static let articleCellIdentifier = "articleCell"
+    
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var tagLabel: UILabel!
+    @IBOutlet weak var autherLabel: UILabel!
+    @IBOutlet weak var subTitleLabel: UILabel!
+    @IBOutlet weak var dispatchButton: UIButton!
+    @IBOutlet weak var moreTagsLabel: UILabel!
+    @IBOutlet weak var addToFavoritesButton: UIButton!
+    
+    var delegate: ArticleTableViewDelegate? = nil
+    
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        self.layer.cornerRadius = 20
+        contentView.layer.cornerRadius = 20
+        contentView.layoutMargins.bottom = 15
+        contentView.layoutMargins.top = 15
+        contentView.layoutMargins.left = 15
+        contentView.layoutMargins.right = 15
+
+        tagLabel.layer.cornerRadius = 10
```

Comment:
Make sure to add text colors, because on older iPhones the default text color can suddenly be different than you expect which results in bugs (e.g. on my iPhone 7 all the texts are white and therefore none of your cells is readable)

Reply:
Still relevant, add the text colors hardcoded and not only through the xcode right pane. When I run the app on my device it looks like this:
![image](https://user-images.githubusercontent.com/100192030/234014072-18969496-249c-476a-8829-3bb607a86588.png)